server/utils/validation.ts

```diff
@@ -92,11 +97,14 @@ export const errorMessage = (field: string, text: string): ErrorMessage => {
   }
 }
 
-const extractValidationErrors = (error: SanitisedError | Error) => {
+const extractValidationErrors = (error: SanitisedError | Error): Record<string, string> | string => {
   if ('data' in error) {
-    if (error.data['invalid-params']) {
+    if (error.data['invalid-params']?.length) {
       return generateErrors(error.data['invalid-params'])
     }
+    if (typeof error.data === 'object' && error.data !== null && 'detail' in error.data) {
```

Comment:
I *think* just the `'detail' in error.data` would be sufficient here but I'd rather be safe than sorry

Reply:
I was getting type errors there, so had to add all this extra gumph